mama/c_cpp/src/cpp/MamaSymbolList.cpp

```diff
@@ -162,6 +163,7 @@ namespace Wombat
 
             return MAMA_STATUS_OK;
         }
+        */
```

Comment:
Wanted to leave it in as wasn't sure if this was  a bug or not - it's part of the same change as the one below. Will raise a new ticket to track.